configuration/phpspx.md

```diff
@@ -0,0 +1,41 @@
+# PHP SPX Profiling
+
+PHP SPX is a simple, free and open source profiler. It covers both FPM and CLI requests, and provides a built-in UI to visualise the generated reports. Of which a [demo can be seen here](https://noisebynorthwest.github.io/php-spx/demo/report.html?key=spx-full-20191229_175636-06d2fe5ee423-3795-233665123).
+
+For more information, please see the [PHP SPX Github Repository](https://github.com/NoiseByNorthwest/php-spx).
+
+
+## Configuration
+
+
+To enable PHP SPX in your project environment, add to your `.env` file:
+```
+WARDEN_PHP_SPX=1
+```
+
+## Usage
+
+### CLI
+
+Running the `warden shell` command, will place you within a PHP-SPX instrumented terminal. After running PHP CLI commands, you should see a SPX report printed after the command finishes. 
```

Comment:
I think you changed this to `warden spx` right?

Reply:
Yup, I'll get this updated to match the changes

Reply:
Appreciate all your work and patience on this!

Reply:
All good! Updated the CLI section and rebased from main